FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,47 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+
+			switch result {
+			case .failure:
+				completion(.failure(Error.connectivity))
+
+			case let .success((data, response)):
+				guard let images = FeedImagesMapper.map(data: data, response: response) else {
+					return completion(.failure(Error.invalidData))
+				}
+				completion(.success(images))
+			}
+		}
+	}
+}
+
+final class FeedImagesMapper {
```

Comment:
Since `FeedImagesMapper` is an implementation detail of `RemoteFeedLoader`, and it's declared in the same file, it should be marked as `private`.
```suggestion
private final class FeedImagesMapper {
```

Reply:
👍